net/tstun/wrap.go

```diff
@@ -820,13 +820,13 @@ func (t *Wrapper) filterPacketOutboundToWireGuard(p *packet.Parsed, pc *peerConf
 			header.ToResponse()
 			outp := packet.Generate(&header, p.Payload())
 			t.InjectInboundCopy(outp)
-			return filter.DropSilently // don't pass on to OS; already handled
+			return filter.DropSilently, gro // don't pass on to OS; already handled
```

Comment:
I was trying to remember the semantics of the gro in vs. gro out on the filter func and looked it up, but missed part in the docs too:

>// nil g for the first packet in a given vector, and continue passing the
// returned *gro.GRO for all remaining packets in said vector. If g is non-nil
// after the last packet for a given vector is passed through the GROFilterFunc,
// the caller must also call g.Flush().
type GROFilterFunc func(p *packet.Parsed, w *Wrapper, g *gro.GRO) (filter.Response, *gro.GRO)

I assume that the call to `g.Flush()` must actually happen on the returned *gro.GRO? Either way we should probably clarify the relationship a little more in the doc string, as I assume that sets clear expectations around this needing to always be returned.

Reply:
will address in a separate PR